design/src/high-level/genre-analysis.md

```diff
@@ -13,67 +13,121 @@
+- partially self-directed goals
+- challenge can be overcome by spending time waiting
+- highly moddable
+- procedurally generated world
+
+## Why is this genre fun?
+
+- puzzle like, but complete solutions are easy to achieve
+- optional optimizations on top of said solutions
+- satisfaction of watching your contraptions run
+- satisfaction of taming the wild
+- interesting, unexpected failures due to unexpected interactions or subtle bugs in the player's design
+- gameplay can be extended, enhanced, or changed through mods or built in options
+- hits on "engineering creativity" in a way that other games don't
+  - engineering creatvitiy is more about coming up with solutions within constraints. It's not a blank canvas. It's based on analytically evaluable metrics
+- scratches similar itches to programming, but with clearer gradients and fewer frustrations
+  - great debugging: can see what's working and why with exceptional visualization
+  - basic solutions are straightforward
+  - very weak syntax constraints, very intuitive syntax
+  - solutions generally do not need to be as robust as in the real world
+  - tech progression doesn't break previous (natural) assumptions
+  - good quality of life tools and docs
 
 ## Adjacent Genres
 
-- Base and city builders
-  - Build out a complex and delightful area, but with no / little automation
+This list is roughly ordered by the closeness to the genre.
+
 - Zachlike puzzles (automation heavy puzzle games)
+  - Opus Magnum, Incredible machine
+  - Greater focus on additional (seconday) optimization constriants (which are provided by the game)
   - Micro-optimization as an entire game
+- Base and city builders
+  - Sim City, The Sims, Cities: Skylines, Dragon Quest Builders
+  - Build out a complex and delightful area, but with no / little automation
+  - Most of the actual functioning of your constructions is abstracted away
 - Colony sims
+  - Rimworld, Dwarf Fortress, Oxygen Not Included
   - Balance complex systems, but with a focus on the needs of your workers, not scaling production
+  - More about the emergent push and pull of life in the colony. Less about steadily pushing towards some goal.
 - Idle games
+  - Forager, Cookie Clicker, Candy box
   - Automate everything, but without the complex chains
+  - Additional mechanics and optimization opportunities unfold over time
 - Train simulators
+  - Open TTD, Mini Metro
   - Focusing almost entirely on logistic networks powered by trains
 - Real Time Strategy
-  - Command units and build bases and extract resources
+  - Starcraft, Warcraft III, Red Alert
+  - Command units, build bases, and extract resources
   - Very little automation: resource production is almost entirely abstracted
   - Much more time pressure!
+- Level builders
+  - Y'know, like Mario Maker
+  - Focused on creating interesting experiences and, in some cases, creating fun and interesting contraptions
+- 5X games
```

Comment:
5X? or 4X?

In 4X games, I would say there is little to no RTS. I think there is little to no city-building either?

One interesting way to characterize might be: "map painting"? The contraption element is almost abstracted away here, although one could argue that the contraption is political/economical in nature.

Reply:
I meant 4X here. Typo typo....

I think your "map painting" analysis makes more sense here, although honestly I think this section can just be cut. You're right- much of the traditional parts of a factory builder have been abstracted away. Will change